openassessment/templates/openassessmentblock/oa_submission_answer.html

```diff
@@ -10,7 +10,7 @@ <h2 class="submission__answer__part__prompt__title">{% trans "The question for t
                 </div>
             </article>
             <div class="submission__answer__part__text">
-                <h2 class="submission__answer__part__text__title">{% trans answer_text_label %}</h2>
+                <h2 class="submission__answer__part__text__title">{{ answer_text_label }}</h2>
```

Comment:
I'm currently evaluating `oa_base` and `oa_rubric.html` as well, based on the following search for other "try to translate a variable" ocurrences:

'''
ericfischer@dhcp-18-189-86-91:~/devstack/src/edx-ora2$ grep -H -r --exclude-dir=edx-ora2 '{% trans [^"]' .
./openassessment/templates/openassessmentblock/oa_base.html:                                    <span class="step__label">{% trans assessment.title %}</span>
./openassessment/templates/openassessmentblock/oa_rubric.html:                <span class="question__title__copy">{% trans rubric_feedback_prompt %}</span>
./openassessment/templates/openassessmentblock/oa_rubric.html:                    placeholder="{% trans rubric_feedback_default_text %}"
'''